packages/lodestar/test/perf/chain/stateCache/stateContextCheckpointsCache.test.ts

```diff
@@ -1,23 +1,33 @@
 import {itBench, setBenchOpts} from "@dapplion/benchmark";
 import {config} from "@chainsafe/lodestar-config/default";
-import {ssz} from "@chainsafe/lodestar-types";
+import {allForks, ssz} from "@chainsafe/lodestar-types";
 import {generateCachedState} from "../../../utils/state";
 import {CheckpointStateCache} from "../../../../src/chain/stateCache";
+import {Checkpoint} from "../../../../../types/phase0";
+import {CachedBeaconState} from "@chainsafe/lodestar-beacon-state-transition";
 
 describe("CheckpointStateCache perf tests", function () {
```

Comment:
Mixup? This changes do not belong to this PR

Reply:
I was getting some errors after changing the `yarn run test:perf` command to `yarn run benchmark`, so I made changes here accordingly.